webapp/src/component/layout/Notifications/TaskAssignedItem.tsx

```diff
@@ -24,7 +22,9 @@ export const TaskAssignedItem: FunctionComponent<TaskAssignedItemProps> = ({
   notification,
   ...props
 }) => {
-  const destinationUrl = `/projects/${notification.project?.id}/task?number=${notification.linkedTask?.number}`;
+  const destinationUrl = `${LINKS.GO_TO_PROJECT_TASK.build({
+    [PARAMS.PROJECT_ID]: notification.project!.id,
+  })}?number=${notification.linkedTask!.number}`;
```

Comment:
This is better, it would be even better if you'd be able to utilize the `getTaskUrl` function, so we have this complicated thing on just one place.

Reply:
I tried, but it is in the ee module which ESLint didn't like that I used it...

Reply:
I put the `getTaskUrl` into the `links.tsx` file.

Is it fine with you?

Reply:
Yep, can be.

Just for info:
The ee module is special, because it is optionaly, so you can only import from the index file `tg.ee`, and there are two variants of the file the `ee` and the `oss`, the oss is basically just a dummy file which will be used when you are running the oss version of Tolgee (without the ee module).